chia/rpc/data_layer_rpc_api.py

```diff
@@ -97,6 +98,13 @@ async def get_value(self, request: Dict[str, Any]) -> Dict[str, Any]:
             hex = value.hex()
         return {"value": hex}
 
+    async def get_keys(self, request: Dict[str, Any]) -> Dict[str, Any]:
+        store_id = bytes32(hexstr_to_bytes(request["id"]))
+        if self.service is None:
+            raise Exception("Data layer not created")
+        keys = await self.service.get_keys(store_id)
+        return {"keys": keys}
```

Comment:
I always forget, are we supposed to explicitly turn the bytes to hex?

```suggestion
        return {"keys": [f"0x{key.hex()}" for key in keys]}
```

I'll note that I really dislike the `0x` prefix.  It always reads to me as indicating a hex-encoded integer.  Is there some other case where people prefix arbitrary blobs with `0x`?  Certainly this is unrelated to this PR and won't hold it up.